imports/ui/pages/admin/AdminPageApprovedPosts.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {AutoForm, LongTextField} from 'uniforms-unstyled';
+import _ from 'underscore';
+import getPosts from '/imports/api/posts/queries/getPost.js'
+
+export default class AdminPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.getPosts = this.getPosts.bind(this);
+        this.logout = this.logout.bind(this);
+        this.state = {
+            posts: [],
+            isApproved: false
+        }
+    }
+
+    componentDidMount() {
+        this.getPosts()
+    }
+
+    getPosts() {
+        Meteor.call("post.lists", true, (err, res) => {
+            console.log(res);
+            if (!err) {
+                this.setState({
+                    posts: res
+                });
+            }
+        });
+    }
+    changeFalse(_id){
+        Meteor.call("changeToFalse", true, _id);
```

Comment:
- Parameters are not given in the correct order
- Name of the method is not correct, adapt to something more intuitive
- From client side we should only send the id of post, not true or false for approved)